_faq/about/03-costs.md

```diff
@@ -6,8 +6,12 @@ tags:
 ---
 ## Registration
 
-Registration for each event will be via a simple Google Form owned by Soc-RSE. All data will be deleted from the form after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors. 
+Registration for each event will be via dedicated registration link on its webpage.
+All data will be deleted from the form after each event. The data will **not** be passed to Zoom or Discord. It will be possible to join the Zoom events via a link and a browser therefore no Zoom account is needed. The link and password to each event will be emailed to the registered participants an hour before the event is due to start.  If you agree for the sponsors to contact you when completing the registration form, the basic information of name and email address will be forwarded to the sponsors.
 
 ## Do I need to pay to attend SORSE events?
 
-No! It’s completely free to the community with the national RSE Groups from many countries collaborating to bring this to you. 
+No! It’s completely free to the community with the national RSE Groups from many countries collaborating to bring this to you.
+
+
+[indico]: https://indico.scc.kit.edu
```

Comment:
I propose additionally linking to the category where we will create all the events in, so that visitors can see what is going to happen when.
You can find it at https://indico.scc.kit.edu/category/95/

Reply:
```suggestion
```
actually this should be removed, as it's not used anywhere in the text :sweat_smile: 
I added the link to the privacy policy (see https://github.com/SORSE/sorse.github.io/pull/229/files#r455623527). Or shall we add it to the text here as well?

Reply:
I think it is sufficient to have the proper link in the privacy policies.